CONTRIBUTING.md

```diff
@@ -2,6 +2,17 @@
 
 # How to contribute to the Tribler project? #
 
+## Checking out the Stabilization Branch ##
+The stabilization branch `next` contains the most up to date bugfixes. If your issue cannot be reproduced there, it is most likely already fixed.
+
+To backup your Tribler installation and checkout the latest version of the stabilization branch, please perform the following steps.
+* Copy the `.Tribler` folder to a safe location on your system (for instance the desktop) Make sure to leave the original folder on its original location. This folder is located at `~/.Tribler/` (Linux/OSX) or `%APPDATA\.Tribler` (Windows).
+* Rename the `tribler` installation folder to `oldTribler`.
```

Comment:
All installers are on 
https://github.com/Tribler/tribler/releases
The Linux one is a .deb, the tar file is just the source code.